### ISSUE TITLE
setting Logged: Yes does not result in field changes being recorded in Action Log

### ISSUE THREAD

**Issue by smutniak** [TexasDigitalLibrary/Vireo#1575]
Logged:Yes not working as described in:

https://texasdigitallibrary.atlassian.net/wiki/spaces/VUG/pages/857112664/Field+Profiles
or
https://texasdigitallibrary.atlassian.net/wiki/spaces/VUG/pages/823885942/Action+Log

From Jon Crossno

**Comment by jcross2562** [comment]
I have set the field properties for some organizations to Logged:Yes in the hopes that changes made in the Admin view would populate in the Action Log. (The documentation indicates that is the purpose of the setting.) As an example, in the attached screenshots I've set the Default Embargoes field to show the Logged = Yes (first screenshot). When viewing the original submission, you can see one embargo option is selected (second screenshot); however, after changing the embargo option, the change is not posted in the Action Log (third screenshot).

Does this help clarify things?

![embargo_properties_logged+yes](https://user-images.githubusercontent.com/127526737/227647850-7e7a1872-d689-4f46-b356-c0fadd8b6f30.png)
![submission-document_info-original](https://user-images.githubusercontent.com/127526737/227647902-9e9b8e68-eb63-4008-838c-1b546cc4fb3d.png)
![submission-document_info-updated](https://user-images.githubusercontent.com/127526737/227647926-50813216-46b8-400e-a348-af560b42f691.png)

**Comment by cstarcher** [comment]
@jcross2562 Was the Logged setting changed before or after the submission was created. If it was changed after, it will not apply to the submission. It will only apply to new submissions created after the field profile was updated. Once a submission is created, the current state of the field profiles is set to that submission and does not change with changes to the field profiles at the organization level.

**Comment by jcross2562** [comment]
@cstarcher Sorry for the delay in responding. I wasn't 100% sure, so I needed to do some testing. And I think I can say now that the Logged setting was changed after the submission was created. I deleted the old test submission and submitted a new one. And I can confirm that the updates are being loaded to the Action Log now. I don't think I was aware that the field profiles won't apply to an existing submission, though.
